src/components/cluster/detail/scale_node_pool_modal.js

```diff
@@ -0,0 +1,439 @@
+        );
+        return {
+          title: `Remove ${workerDelta} worker node${this.pluralize(
+            workerDelta
+          )}`,
+          style: 'danger',
+          disabled: !this.state.scaling.maxValid,
+        };
+      }
+
+      if (this.state.scaling.min != this.props.cluster.scaling.min) {
+        return {
+          title: 'Apply',
+          style: 'success',
+          disabled: !(
+            this.state.scaling.minValid && this.state.scaling.maxValid
+          ),
+        };
+      }
+
+      if (this.state.scaling.max != this.props.cluster.scaling.max) {
+        return {
+          title: 'Apply',
+          style: 'success',
+          disabled: !(
+            this.state.scaling.minValid && this.state.scaling.maxValid
+          ),
+        };
+      }
+
+      return {
+        disabled: true,
+      };
+    }
+
+    if (workerDelta === 0) {
+      return {
+        disabled: true,
+      };
+    }
+
+    if (workerDelta > 0) {
+      return {
+        title: `Add ${workerDelta} worker node${pluralizeWorkers}`,
+        style: 'success',
+        disabled: !(this.state.scaling.minValid && this.state.scaling.maxValid),
+      };
+    }
+
+    if (workerDelta < 0) {
+      return {
+        title: `Remove ${Math.abs(workerDelta)} worker node${pluralizeWorkers}`,
+        style: 'danger',
+        disabled: !(this.state.scaling.minValid && this.state.scaling.maxValid),
+      };
+    }
+
+    return {
+      disabled: true,
+    };
+  };
+
+  render() {
+    var warnings = [];
+    if (
+      this.state.scaling.max < this.props.workerNodesRunning &&
+      this.state.scaling.minValid
+    ) {
+      var diff = this.props.workerNodesRunning - this.state.scaling.max;
+
+      if (
+        this.supportsAutoscaling(
+          this.props.provider,
+          this.props.cluster.release_version
+        )
+      ) {
+        warnings.push(
+          <CSSTransition
+            classNames='rollup'
+            enter={true}
+            exit={true}
+            key={1}
+            timeout={this.rollupAnimationDuration}
+          >
+            <p key='node-removal'>
+              <i className='fa fa-warning' /> The cluster currently has{' '}
+              {this.props.workerNodesRunning} worker nodes running. By setting
+              the maximum lower than that, you enforce the removal of{' '}
+              {diff === 1 ? 'one node' : diff + ' nodes'}. This could result in
+              unscheduled workloads.
+            </p>
+          </CSSTransition>
+        );
+      } else {
+        warnings.push(
+          <CSSTransition
+            classNames='rollup'
+            key={2}
+            timeout={this.rollupAnimationDuration}
+          >
+            <p key='node-removal'>
+              <i className='fa fa-warning' /> You are about to enforce the
+              removal of {diff === 1 ? 'one node' : diff + ' nodes'}. Please
+              make sure the cluster has enough capacity to schedule all
```

Comment:
in this we're scaling a nodepool, does this warning still make sense? ping @marians as well